File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/SeatQuantity.cs
using System;

namespace ConferenceManagement
{
    public class SeatQuantity
    {
        public SeatQuantity(Guid seatTypeId, int quantity)
        {
            SeatTypeId = seatTypeId;
            Quantity = quantity;
        }

        public Guid SeatTypeId { get; private set; }
        public int Quantity { get; private set; }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/SeatQuantity.java
package com.microsoft.conference.management.domain.Models;

public class SeatQuantity {
    public String SeatTypeId;
    public int Quantity;

    public SeatQuantity(String seatTypeId, int quantity) {
        SeatTypeId = seatTypeId;
        Quantity = quantity;
    }
}
